### PR TITLE
EN 1279 delete category

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: test/ui/categories/common/DeleteCategoryModal.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+describe('DeleteCategoryModal', () => {
+  let component;
+  beforeEach(() => {
+    component = shallow(<DeleteCategoryModal />);
+  });
+
+  describe('component with required props only', () => {
+    it('renders without crashing', () => {
+      expect(component.exists()).toEqual(true);
```
Comment (fedelocci) [inline test/ui/categories/common/DeleteCategoryModal.test.js]: ToBe

File: test/ui/categories/common/DeleteCategoryModalContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    it('maps info property from state', () => {
+      expect(mapStateToProps(TEST_STATE))
+        .toHaveProperty('info', MODAL_INFO);
+    });
+  });
+
+  describe('mapDispatchToProps', () => {
+    it('should map the correct function properties', () => {
+      const props = mapDispatchToProps(dispatchMock);
+      expect(props.onConfirmDelete).toBeDefined();
```
Comment (fedelocci) [inline test/ui/categories/common/DeleteCategoryModalContainer.test.js]: use toHaveProperty

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by mcarta-rt** [comment]
please retest this

**Comment by mcarta-rt** [comment]
please retest this

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
All is well

**Pull request merged**